test/fail_compilation/diag11769.d

```diff
@@ -2,9 +2,9 @@
 TEST_OUTPUT:
 ---
 fail_compilation/diag11769.d(18): Error: diag11769.foo!string.bar called with argument types (string) matches both:
-fail_compilation/diag11769.d(13):     diag11769.foo!string.bar(wstring _param_0)
+fail_compilation/diag11769.d(13):     diag11769.foo!string.bar(wstring _param_0) 
 and:
-fail_compilation/diag11769.d(14):     diag11769.foo!string.bar(dstring _param_0)
+fail_compilation/diag11769.d(14):     diag11769.foo!string.bar(dstring _param_0) 
```

Comment:
Having to add an extra space at the end of plenty of test should be a red flag that something is wrong